README.rst

```diff
@@ -0,0 +1,301 @@
+| ICL (Ice Lake)       | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| JSL (Jasper Lake)    | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| EHL (Elkhart Lake)   | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| TGL (Tiger Lake)     | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| DG1 (Xe MAX)         | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| SG1                  | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| RKL (Rocket Lake)    | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ADL-S (Alder Lake S) | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ADL-P (Alder Lake P) | VPL          |                 | ✔               |                 | ✔               |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| Future platforms...  | VPL          |                 | ✔               |                 | ✔               |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| Multi GPU            | Legacy + VPL | See above                         | ✔               | via env var [2] |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+
+**Notes**:
+
+* [1] KBLx is one of: KBL (Kaby Lake), CFL (Coffe Lake), WHL (Whiskey Lake), CML (Comet Lake), AML (Amber Lake)
+* [2] On the multi GPU system which has both "Legacy" and "VPL" GPUs Media SDK Dispatcher loads Media SDK Legacy
+  Runtime (libmfxhw64.so.1) by default. VPL Runtime (libmfx-gen.so.1) can be loaded if the following environment
+  variable will be specified::
+
+    export MSDK_RUNTIME=libmfx-gen.so.1
```

Comment:
See #2728: we need to rename env. variable to `INTEL_MEDIA_RUNTIME` which should have 2 possible values: `MSDK|ONEVPL`.

Reply:
Done